### PR TITLE
Improve Release Instructions

### PR DESCRIPTION
**Pull request by ericherman**
Adding some notes about the process.

-----
[View rendered CONTRIBUTING.md](https://github.com/publiccodenet/standard/blob/eh-releasing-20201026/CONTRIBUTING.md)
[View rendered README.md](https://github.com/publiccodenet/standard/blob/eh-releasing-20201026/README.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,6 +46,34 @@ Your ideas, documentation and code have become an integral part of this project.
...
+    - [ ] Update [`AUTHORS.md`](AUTHORS.md) with new contributors
+    - [ ] Update [`CHANGELOG.md`](CHANGELOG.md)
+    - [ ] Perform extra pass on diff to the 'master' branch
+        - Reread any section or paragraph to ensure wording changes still fit the whole and do not contain grammar or spelling errors
+        - If needed, commit changes and repeat extra pass
+    - [ ] Push branch, open a pull request to the 'master' branch
+        - Request review from multiple reviewers, especialy a proofreader
+    - [ ] Generate new PDF of the primary content
+        - Serve html content with `script/serve.sh`
+        - In a browser, navigate to http://127.0.0.1:4000/
```
Comment (bvhme) [inline CONTRIBUTING.md:66]:
```suggestion
        - You can validate the server is running by navigating to http://127.0.0.1:4000/ in your browser
```

Comment (ericherman) [reply CONTRIBUTING.md]: Ah, well, I just pushed a similar change. Let me know which wording you prefer.

**Comment by bvhme** [comment]
Looks good, to merge even, even though this is still a _draft_ pull request. I don't know if it's necessary to start the list with 0.

**Comment by ericherman** [comment]
> Looks good, to merge even, even though this is still a _draft_ pull request.

We could certainly merge it as-is. I imagine we'll be adding more to it as we finish out the release process. I note that we do not yet have instructions for creating the PDF covers, the tagging, and making available as a github release. That said, all of that could be in future merge requests.

> I don't know if it's necessary to start the list with 0.

Ah yes, that is needlessly geeky. I'll make a commit which renumbers the steps.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,6 +46,34 @@ Your ideas, documentation and code have become an integral part of this project.
...
+        - Serve html content with `script/serve.sh`
+        - Optionally, for a visual pre-check, navigate to http://127.0.0.1:4000/ in a browser
+        - Generate the pdf with `script/pdf.sh`
+    - [ ] Create new cover for print
+    - [ ] Create new cover for web PDF
+    - [ ] Create PDF for GitHub release with front and back cover included
+4. Send the files for print to the printer
+    - [ ] Cover file
+    - [ ] Inside pages PDF
+5. Create GitHub release with the release notes and version number
```
Comment (bvhme) [inline CONTRIBUTING.md:74]: This point involves creating a Git tag, which should be named the exact version number.

After this the release branch needs to be merged with the `main`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -46,6 +46,34 @@ Your ideas, documentation and code have become an integral part of this project.
...
+    - From 'develop', git checkout -b "release-$MAJOR.$MINOR.$PATCH"
+3. Update the new release
+    - [ ] Update version number in `_config.yml` and `README.md`
+    - [ ] Update [`AUTHORS.md`](AUTHORS.md) with new contributors
+    - [ ] Update [`CHANGELOG.md`](CHANGELOG.md)
+    - [ ] Perform extra pass on diff to the 'master' branch
+        - Reread any section or paragraph to ensure wording changes still fit the whole and do not contain grammar or spelling errors
+        - If needed, commit changes and repeat extra pass
+    - [ ] Push branch, open a pull request to the 'master' branch
+        - Request review from multiple reviewers, especialy a proofreader
```
Comment (Ainali) [inline CONTRIBUTING.md:63]: Ironically, there's a typo in this sentence: especialy -> especially.

**Comment by ericherman** [comment]
rebased

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -46,6 +46,45 @@ Your ideas, documentation and code have become an integral part of this project.
 
 In fact, feel free to open a PR to add your name to the [`AUTHORS`](AUTHORS.md) file and get eternal attribution.
 
+## Releasing a new version
+
+1. Review state of the 'develop' branch
+    - Ensure all changes intended for release are merged
+    - Invite a proofread of the current state of the branch
+2. Create a release branch
+    - From 'develop', git checkout -b "release-$MAJOR.$MINOR.$PATCH"
```
Comment (bvhme) [inline CONTRIBUTING.md:55]:
```suggestion
    - From 'develop', `git checkout -b "release-$MAJOR.$MINOR.$PATCH"`
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -46,6 +46,45 @@ Your ideas, documentation and code have become an integral part of this project.
...
+4. Send the files for print to the printer
+    - [ ] Cover file
+    - [ ] Inside pages PDF
+5. Create GitHub release with the release notes and version number
+    - [ ] `git tag $MAJOR.$MINOR.$PATCH`
+    - [ ] `git push --tags`
+    - [ ] from https://github.com/publiccodenet/standard/releases edit the tag
+        - Title the release
+        - Add changelog bullets
+        - Drag-and-drop the generated .pdf into the assets
```
Comment (bvhme) [inline CONTRIBUTING.md:85]: You can also do this from the GitHub releases feature and it'll create a tag for you.

**Review by bvhme — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (ericherman) [reply CONTRIBUTING.md:85]: We've doc'd it as-done, when we change, let's update the docs to match.

**Pull request merged**